packages/release/proxy-env

```diff
@@ -0,0 +1,10 @@
+{{#if settings.network.http-proxy}}
+HTTP_PROXY={{settings.network.http-proxy}}
+http_proxy={{settings.network.http-proxy}}
+{{/if}}
+{{#if settings.network.https-proxy}}
+HTTPS_PROXY={{settings.network.https-proxy}}
+https_proxy={{settings.network.https-proxy}}
+{{/if}}
+NO_PROXY={{#each settings.network.no-proxy}}{{this}},{{/each}}localhost,127.0.0.1,169.254.169.254,169.254.170.2,/var/run/docker.sock{{#if settings.kubernetes.api-server}},{{host settings.kubernetes.api-server}},.svc,.default,.local,.cluster.local{{/if}}
```

Comment:
My only remaining concern is that we list /var/run/docker.sock but not /run/docker.sock, even though our docker.socket unit lists /run/docker.sock.  Is there a chance we wouldn't proxy requests because we're using one instead of the other, even though they're symlinked?  Do we need to list both?

Reply:
All I know is ECS agent explicitly dials `/var/run/docker.sock`. As for how that gets resolved is unsure to me. I do know from testing that the ECS variant currently works with what's currently there.

Reply:
@samuelkarp any thoughts?

Reply:
I haven't seen any problems with using `/var/run/docker.sock` on other platforms with symlinks (like Amazon Linux 2).

We should have test coverage for the ECS agent in the agent's test suite, so I'm comfortable relying on that for validation.

Reply:
I know that we're seeing the basics work, I guess my concern is that I'm not sure whether there could be any other components called by Docker (or whatever) that are assuming `/run/docker.sock`, and they've just worked for us because of the symlink, but suddenly they'd fail (or do horrible things) because they're being proxied.  (I don't know of anything, and we can fix it if we found it, hence my ship-it)